webview/src/experiments/components/table/Indicators.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { MouseEventHandler, ReactNode } from 'react'
+import { SortDefinition } from 'dvc/src/experiments/model/sortBy'
+import cx from 'classnames'
+import { MessageFromWebviewType } from 'dvc/src/webview/contract'
+import styles from './styles.module.scss'
+import { Icon } from '../../../shared/components/Icon'
+import SvgSortPrecedence from '../../../shared/components/icons/SortPrecedence'
+import SvgFilter from '../../../shared/components/icons/Filter'
+import { sendMessage } from '../../../shared/vscode'
+
+const Indicator = ({
+  children,
+  count,
+  'aria-label': ariaLabel,
+  onClick
+}: {
+  children: ReactNode
+  count?: number
+  'aria-label'?: string
+  onClick?: MouseEventHandler
+}) => (
+  <button
+    className={cx(styles.indicatorIcon, count && styles.indicatorWithCount)}
+    aria-label={ariaLabel}
+    onClick={onClick}
+  >
+    {children}
+    {count ? <span className={styles.indicatorCount}>{count}</span> : null}
+  </button>
+)
+
+const focusFiltersTree = () =>
+  sendMessage({ type: MessageFromWebviewType.FOCUS_FILTERS_TREE })
+const focusSortsTree = () =>
+  sendMessage({ type: MessageFromWebviewType.FOCUS_SORTS_TREE })
+
+export const Indicators = ({
+  sorts,
+  filters
+}: {
+  sorts?: SortDefinition[]
+  filters?: string[]
+}) => {
+  return (
+    <div className={styles.tableIndicators}>
+      <Indicator
+        count={sorts?.length}
+        aria-label="sorts"
+        onClick={focusSortsTree}
+      >
+        <Icon width={16} height={16} icon={SvgSortPrecedence} />
```

Comment:
There is an `AllIcon` const with all the icons in there (needs to add each one manually). I added this because it was an easy way to see all the available icons, but importing them directly like you did is way more convenient. The only thing I would say about this is that it'd be better to import it as `SortPrecedence` from the `icons/index.ts` file. Or we could change the SVGR script not to generate an index file, I don't really mind. I'll do a follow up removing `AllIcons` and importing the icons directly.